src/views/Store.js

```diff
@@ -300,15 +303,53 @@ const BuyPage = () => {
             <>
               Dentec Comfort-Air refill pack of 16 pairs of{" "}
               <strong>N95 Filters</strong>, and box of{" "}
-              <strong>100 alcohol-free wipes</strong>.
+              <strong>100 alcohol-free respirator wipes</strong>.
             </>
           }
           dataSheetUrl={DentecData}
           price="$99.99"
-          buyBtnId="buy-3m-refill"
+          buyBtnId="buy-dentec-refill"
```

Comment:
Nice catch

Reply:
Good idea.  Will "Sqoosh" and push new commit with smaller files.